tfx/examples/chicago_taxi_pipeline/taxi_pipeline_kubernetes.py

```diff
@@ -0,0 +1,182 @@
+# Lint as: python2, python3
```

Comment:
Because this is not yet a suggested way to run things, can you move this to `tfx/orchestration/experimental/kubernetes/examples`?

Reply:
Done.